src/waveform/renderers/waveformrenderersignalbase.cpp

```diff
@@ -223,3 +226,13 @@ void WaveformRendererSignalBase::getGains(float* pAllGain, float* pLowGain,
         }
     }
 }
+
+float* WaveformRendererSignalBase::unscaleTable() {
```

Comment:
```suggestion
std::span<float, 256> WaveformRendererSignalBase::unscaleTable() {
```

Reply:
or const std::array<float, 256>& maybe? what do you prefer?

Reply:
I'd prefer `std::span` tbh, at least until a `static constexpr` table is an option (which it is not until constexpr std::pow is available). A reference to a value type when there is also a type to represent that exact concept type available seems like the inferior solution.

Reply:
done